alpha/apps/kaltura/lib/kPlaybackContext.php

```diff
@@ -60,6 +65,22 @@ public function setFlavorAssets($flavorAssets)
 		$this->flavorAssets = $flavorAssets;
 	}
 
+	/**
+	 * @return array
+	 */
+	public function getCaptionAssets()
+	{
+		return $this->captionAssets;
+	}
+
+	/**
+	 * @param array $captionAssetsItems
+	 */
+	public function setCaptionAssets($captionAssetsItems)
```

Comment:
rename captionAssetsItems to captionAssets since it has a different meaning that can confuse